docs/source/specs/openapi.yml

```diff
@@ -502,6 +502,45 @@ paths:
             $ref: '#/definitions/Error'
       security:
       - api_key: []
+  /reports/costs/ocp/:
+    get:
+      tags:
+        - Report
+      summary: Query to obtain cost reports
+      operationId: getCostReports
```

Comment:
<img width="550" alt="image" src="https://user-images.githubusercontent.com/29379759/52877070-f4f92e00-3126-11e9-9dea-d3b41be5a45d.png">

Should update the operationId of this and the AWS report.